community_supplied/Security/test-secure-boot-status.rule

```diff
@@ -0,0 +1,103 @@
+/*
+ * This rule checks if secureboot is running and enforced and returns
+ * an error if it is not.
+ *
+ * This rule relies on a file present in 19.3R1 and beyond.
+ * This rule is only applicible to devices where secureboot is capable.
+ * If this rule is run on a device that is incapable of secureboot,
+ * the result will be "secureboot status is unknonw" and the rule should be
+ * removed.
+ */
+healthbot {
+    topic security {
+        rule check-secureboot-status {
```

Comment:
If possible, please add rule description and synopsis. So it provides information about rule functionality. BTW comments provide information about rule but comments disappear after loading rule in HB MGD. Hence requesting you to add description and synopsis